src/Panel/User.php

```diff
@@ -30,15 +30,32 @@ public function breadcrumb(): array
     }
 
     /**
-     * Panel icon definition
+     * Panel icon defaults
      *
-     * @param array $params
      * @return array
      */
-    public function icon(array $params = null): array
+    public function iconDefaults(): array
     {
-        $params['type'] = 'user';
-        return parent::icon($params);
+        return [
+            'type'  => 'user',
+            'ratio' => '1/1',
+            'back'  => 'black',
+            'color' => '#c5c9c6',
```

Comment:
Is it possible to use our CSS vars for the colors? This would allow changing these colors with the general color scheme in a custom `panel.css`.

Reply:
With the changes on my PR this would actually work. With this PR alone, no.